rules/no-missing-di.js

```diff
@@ -0,0 +1,98 @@
+const {
+  is
+} = require('bpmnlint-utils');
+
+/**
+ * A rule that checks that there is no BPMNDI information missing for elements,
+ * which require BPMNDI.
+ */
+module.exports = function() {
+
+  function check(node, reporter) {
+
+    if (!is(node, 'bpmn:Definitions')) {
+      return;
```

Comment:
If you would like to stop traversing, returning false from a check is your friend, cf. https://github.com/bpmn-io/bpmnlint/issues/53#issuecomment-752262159.

You could also, alternatively collect all drawable elements, as the lib traverses the document. Either way works for me.

Reply:
Very good point, and lovely that it is as easy as returing `false`